TwitchLib.EventSub.Core/Models/ChannelPoints/AutomaticRedemptionReward.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace TwitchLib.EventSub.Core.Models.ChannelPoints;
+/// <summary>
+/// Basic information about the reward that was automatically redeemed, at the time it was redeemed.
+/// </summary>
+public sealed class AutomaticRedemptionReward
+{
+
+    /// <summary>
+    /// The type of reward. One of:
+    /// ssingle_message_bypass_sub_mode
+    /// send_highlighted_message
+    /// random_sub_emote_unlock
+    /// chosen_sub_emote_unlock
+    /// chosen_modified_sub_emote_unlock
+    /// </summary>
+    public string Type { get; set; } = string.Empty;
+    /// <summary>
+    /// The reward cost.
+    /// </summary>
+    public int Cost { get; set; }
+
+    /// <summary>
+    /// Optional. Emote that was unlocked.
+    /// </summary>
+    public UnlockedEmote? UnlockedEmote { get; set; } = new();
```

Comment:
if it can be null, the default value is not set
```suggestion
    public UnlockedEmote? UnlockedEmote { get; set; }
```